src/components/wallet/TokenAmount.js

```diff
@@ -3,9 +3,12 @@ import { formatTokenAmount } from '../../utils/amounts'
 
 const FRAC_DIGITS = 5
 
-const TokenAmount = ({ token: { balance, decimals, symbol}, className }) => (
+const TokenAmount = ({ token: { balance, decimals, symbol }, className }) => (
     <div className={className} title={showFullAmount(balance, decimals, symbol)}>
-        {balance && formatToken(balance, decimals)}
+        {balance
+            ? formatToken(balance, decimals)
```

Comment:
:nit: It would be easier to follow this file if helper methods were defined before they are used (especially when they are very similarly named to things we import, and/or wrap things that we import)